mapping/sustainability.yaml

```
@@ -1230,27 +1221,27 @@
     title: Carbon footprint
     module: Climate finance
     indicator: Long-term project impact
-    disclosure format: Disclose the carbon footprint of the project \[value, tons CO2 equivalent, free text to describe the methodology used to measure the carbon footprint\]
+    disclosure format: Disclose the carbon footprint of the project, informing the calculation, the methodology applied, and where the calculation can be found. \[value, tons CO2 equivalent, free text to describe the methodology used to measure the carbon footprint\]
     mapping: |-
         Project-level:
 
-        1. Add a `Metric` object to the `.metrics` array.
+        1. Add a [`Metric`](../../reference/schema.md#metric) object to the [`.metrics`](project-schema.json,,metrics) array.
```

```suggestion
        1. Add a [`Metric`](../../reference/schema.md#metric) object to the [`metrics`](project-schema.json,,metrics) array.
```

plus another "Project-level" which should be "Project level" above

---

Fixed in 73bc7abfdb6a4bf562298f1f80ac2d6cfd7656ec